### ISSUE TITLE
hey dont know were to post this.

### ISSUE THREAD

**Issue by ReK2Fernandez** [BlackArch/blackarch-iso#18]
@noptrix I created a fork of this and making a live cd with the option to have calamares do the actual instalation for my community members(hispagatos), is in early/alpha stage, but I was wondering that maybe you guys want to make it part of the official options.. I can do a PR when I am done.. I can also be the mainteiner of this livecd, will only need maybe some art for the install process as of now is just raw default calamares. 

Basically the way it works is the same for now.. 
users get in and choose to use the default install or type blackarch-install-gui or similar.. for graphical install..

**Comment by ChrisFernandez** [comment]
well nobody reply.. :(  so going to close.

**Comment by heichblatt** [comment]
Hi @ChrisFernandez,

sorry it took us so long to get back to you. I would be interested in your work so far, where can I take a look at it?

**Comment by UndeadHeavy** [comment]
Is Blackarch can be install on gui?

**Comment by UndeadHeavy** [comment]
I need help to install blackarch on uefi gpt pc

**Comment by ReK2Fernandez** [comment]
hey @heichblatt sorry I missed your question, got buried between work notices and my personal projects, I started a new job in December so had to move and dedicate more time to the new position, so had it to a bootable point and never continued, it had lots of bugs more related to the graphical installer "calamares" I may restart it if someone else have not yet taken over the project.

**Comment by ikstream** [comment]
I started to work on a dialog installer, which would be appropriate for a net-install iso, while I think calamares is way more suited for a live iso.

**Comment by PedroSFreitas** [comment]
I agree with @ikstream and I say more, we could have both in the same ISO.
People could choose between the terminal (dialog) installer and the GUI (calamares) one. If we could provide both, we would make more people comfortable of using BlackArch.

**Comment by noptrix** [comment]
any updates on this?

**Comment by PedroSFreitas** [comment]
@noptrix, let's leave this open for now. I am working on something similar (Calamares) and as soon as I finish it I'll port to BlackArch as well.
Unfortunately I don't have an ETA for you, but shouldn't be more than one month or so.
